proposals/cri-o.adoc

```diff
@@ -0,0 +1,220 @@
+== CRI-O
+
+*Name of project*: cri-o
+
+*Description*:
+
+CRI-O is a stable container runtime specifically tailored for Kubernetes.
+A founding principal was not "reinventing the wheel" and so the project uses shared components, refined approaches tested in production, and existing battle-tested code.
+Further, it is tuned for performance, stability, compatibility, and adherence to standards, particularly the Kubernetes Conformance tests.
+In short CRI-O is a building block of a Kubernetes cluster, to facilitate the life cycle of containers as required by the Kubernetes Container-Runtime-Interface (CRI).
+
+CRI-O was born in kubernetes-incubator and grew to production stable there.
+Some have claimed since being a kubernetes project that it is already "a CNCF project", but it is often left as an after thought in brand, awareness and support as a top-level project.
```

Comment:
Where is it you want the brand and awareness to go? Is there some form of support you are looking for as a top level project?

For example, when Helm shifted out of Kubernetes there was a need for help organizing Helm conferences (currently prepping for a second) and it had many contributors (it's the 3rd most contributed to CNCF project) where support for scaling that was needed.

Is cri-o looking to be more than a container runtime for k8s? Is there something else going on?

I ask these questions as I'm curios to understand more of where cri-o is going.

Reply:
I'm glad that kubernetes-incubator was renamed kubernetes-sigs, but cri-o has outgrown that level of incubation.
And truthfully due to not getting the same level of press as containerd or rkt, I get feedback that folks infer it is not considered on the same level.
No cri-o conferences needed. :smiley: